### PR TITLE
Fix: adjust font optimization

### LINKED ISSUE #1785: Troubleshoot typography loading

### PR DESCRIPTION
**Pull request by robhyrk**
fixes #1785

### CODE REVIEW

**Review by Robiquet — APPROVED** [review]

**Review by Robiquet — CHANGES_REQUESTED** [review]
Sorry... just noticed that fonts aren't loading here (Get started modal). Probably because it's using some sort of portal so that the modal is outside of the element where the fonts were added
image

File: layouts/DefaultLayout.tsx [hunk 1]
```diff
@@ -47,24 +46,16 @@ const DefaultLayout: FC<PropsWithChildren> = ({ children }) => {
 
   return (
     <div
-      className={`relative min-h-screen justify-evenly overflow-x-hidden ${
+      className={`${inter.variable} ${kanit.variable} ${
```
Comment (yornaath) [inline layouts/DefaultLayout.tsx:49]: I would probably put this on the body element in _document. Should fix the issue @Robiquet reported

**Review by yornaath — CHANGES_REQUESTED** [review]

**Comment by robhyrk** [comment]
The nextjs docs didn't explain it the best but I found a workaround because for some reason font optimization doesn't load when called in `_document` so I had to add in` _app`, but then there was the issue of not being able to access the `body` tag directly so I ended up adding in the fonts dynamically.

**Comment by Robiquet** [comment]
I'm still seeing the shifting like we have on staging, I wonder if we can move the modal portal to it get the fonts?

**Review by Robiquet — APPROVED** [review]

**Review by yornaath — APPROVED** [review]

**Pull request merged**